kubernetes/README.md

```diff
@@ -68,7 +70,7 @@ except ApiException as e:
 
 ## Documentation for API Endpoints
 
-All URIs are relative to *https://localhost*
+All URIs are relative to *http://localhost*
```

Comment:
it should be http, instead of https before switching to operanapi generator, right?

Reply:
I think `https` was generated from the old swagger-codegen version that we were using.